packages/sanity/src/desk/components/paneItem/PaneItem.tsx

```diff
@@ -115,7 +115,14 @@ export function PaneItem(props: PaneItemProps) {
     [ChildLink, id]
   )
 
-  const handleClick = useCallback(() => setClicked(true), [])
+  const handleClick = useCallback((e: React.MouseEvent<HTMLElement>) => {
```

Comment:
smol nit, but implementation could be simplified to `setClicked(!e.metaKey)`

Maybe also add a little comment about why we do it?